worksheet_B/Mandelbrot.cpp

```diff
@@ -17,22 +136,42 @@ int main()
 	// Generate the image
 	for (int pixelY = 0; pixelY < image.height(); pixelY++)
 	{
-		// TODO: Map the y coordinate into the range minY to maxY
-		//double y0 =
+		//Map the y coordinate into the range minY to maxY
+		double y0 = (pixelY / 800.0) * (maxY - minY) + minY;
 
-		for (int pixelX = 0; pixelX < image.width(); pixelX++)
+		for (int pixelX = 0.0; pixelX < image.width(); pixelX++)
```

Comment:
`pixelX` is an `int` -- assigning `0.0` to it doesn't make sense (the compiler just rounds it to integer `0` anyway)